TicketBooking/Helpers/CustomerDB.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Data.SqlClient;
+using TicketBooking.Models;
+
+namespace TicketBooking.Helpers
+{
+    internal class CustomerDB
+    {
+        public static Customer Get(string customerId)
+        {
+            SqlConnection connection = new SqlConnection();
```

Comment:
1. can we properly Dispose the SqlConnection object. we can use Using
2. Can we implement try catch for this.